src/content/docs/apm/agents/ruby-agent/troubleshooting/update-deprecated-api-calls.mdx

```diff
@@ -13,11 +13,24 @@ redirects:
 
 ## Problem
 
-In New Relic Ruby agent version [4.0.0](/docs/releases/ruby), a number of APIs that had been deprecated for some time were deleted. If you use any of the methods, you must update your API with the recommended replacement before upgrading to Ruby agent version 4.0.0 or higher.
+We occasionally remove deprecated APIs. If you use any of the following methods, you must update your API call(s) with the recommended replacement(s) before upgrading to certain Ruby agent versions.
```

Comment:
I think it's just a style preference, but using the plurals instead of `(s)` seems fine to me 🤷 

It seems like that might also be easier for the translation engines to appropriately handle. Maybe @akristen could chime in here?

Reply:
I agree. [Changed](https://github.com/newrelic/docs-website/pull/11408/commits/03ee45531a09c8ccb8f3f13a1b099dc2dd12cd15).